kyo-prelude/shared/src/test/scala/kyo/debug/DebugTest.scala

```diff
@@ -157,10 +157,10 @@ class DebugTest extends Test:
         "with Stream" in
             testOutput(
                 "DebugTest.scala:53:36",
-                "2147483647",
```

Comment:
@fwbrasil I'm not sure where this number came from (I assume from `Ack.Continue(n)`?), but do these changes make sense?

Reply:
It’s probably Int.MaxValue for ack without a chunk limit. It’s fine to update the assertion